### PR TITLE
Updates/dec 27 kajal

### PR DESCRIPTION
**Pull request by kkaajjaall**
Referee form and employee-information form is done and employee-address form is remain

### CODE REVIEW

File: resources/views/referee-form.blade.php [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        <div class="col-12 mt-5 mb-5 text-center">
+            <h2>Referee Information</h2>
+        </div>
+        <div class="container">
+            <form class="row g-3 needs-validation" method="post" action="{{ $actionUrl }}">
+                @csrf
+                {{-- update code --}}
+                @isset($user)
+                    {{ method_field('PUT') }}
+                    {{-- <input type='hidden' name='referee_id' value='{{ $referee->id }}'> --}}
```
Comment (sumedh-mohod) [inline resources/views/referee-form.blade.php:15]: remove the comment

**Review by sumedh-mohod — COMMENTED** [review]
format code

File: app/Http/Controllers/LocalAddressController.php [hunk 2]
```diff
@@ -33,37 +33,29 @@ public function store(Request $request)
...
-    // public function show(Request $request)
-    // {
-    //     $local_address = LocalAddressData::where('user_id', $request->id)->get()->first();
-    //     $user = User::find($request->id);
-    //     return view('address', compact('local_address', 'user'));
-    // }
-
     public function show($id)
     {
         $local_address =  LocalAddressData::where('user_id', $id)->get()->first();
```
Comment (sumedh-mohod) [inline app/Http/Controllers/LocalAddressController.php:40]: variable name shouldnt have _ and second letter should start with capital letter

**Comment by ithqindia** [comment]
Merged in https://github.com/ithqindia/metroclub/pull/26

**Pull request closed without merging**